odoo/addons/base/res/res_config.py

```diff
@@ -478,6 +458,9 @@ def _get_classified_fields(self):
 
         return {'default': defaults, 'group': groups, 'module': modules, 'other': others}
 
+    def get_default_fields(self, fields):
```

Comment:
The method name and API is not so good, IMHO:
* Why "default" fields? What you ask for is the current value of the stuff, shown as fields in the model. The values are fed as default values in the wizard, but that's an implementation trick.
* Why "fields"? What you ask for are configuration parameters, and such things.
* Why passing a list of fields? Its value is *never* used.

I would further simplify the API of both methods to something like
```python
def get_values(self):
    return {}

def set_values(self):
    pass
```